ompi/attribute/attribute.c

```diff
@@ -50,6 +52,8 @@
  *   means writing a pointer to an instance of something; changing the
  *   value of that instance will make it visible to anyone who reads
  *   that attribute value).
+ * - C also internally store some int attributes of a MPI_Win by value,
+ *   and these attributes appear to be read-only (e.g. set once for all)
```

Comment:
They **are** read-only (they don't just **appear** to be read-only 😄).  And it would be "i.e.," not "e.g."  😈